src/agent/executor/base.ts

```
@@ -0,0 +1,87 @@
+      OpenAIBaseExecutor.log.info`execute ${this.taskType} with response: ${response}`;
+
+      const content = response.choices?.[0]?.message?.content ?? '';
+
+      if (!content.trim()) {
+        throw new Error(`Empty response for ${this.taskType}`);
+      }
+
+      return content;
+    } catch (error) {
+      throw error;
+    }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)